cli/src/org/partiql/cli/Repl.kt

```diff
@@ -41,6 +41,8 @@ private enum class ReplState {
     /** Reading a PartiQL query. Transitions to execute when one of the execution tokens is found. */
     READ_PARTIQL,
 
+    LAST_PARTIQL_LINE,
```

Comment:
Can we please add a comment here explaining the intent of this state, just like the comments on the existing states.

Reply:
Good call - done!